setup.py

```diff
@@ -16,6 +16,7 @@
     "plone.base",
     "plone.i18n",
     "plone.memoize",
+    "plone.openid",
```

Comment:
Do not add this.
```suggestion
```

Reply:
Oh, sorry 🤦🏾 I've been configuring so many projects that I don't remember what I did on each of them 😆 

Will do, thanks!